worked-solutions/01_penguin_classification_solutions.ipynb

```diff
@@ -850,8 +844,8 @@
       "        [3.3600e+01, 1.1300e+01, 2.0000e+03, 2.1100e+02, 1.0000e+00]])\n",
       "\n",
       "Raw output:\n",
-      "tensor([[0.0035, 0.0021, 0.9943],\n",
-      "        [0.9547, 0.0013, 0.0440]])\n",
+      "tensor([[0.0059, 0.0018, 0.9923],\n",
+      "        [0.5084, 0.0350, 0.4567]])\n",
```

Comment:
Predictive performance has worsened quite a bit on the second of these.

Reply:
Not sure that matters if the input isn't "real"? Happy to revert :)